src/scss/components/menu-toggle/menu-toggle.stories.ts

```diff
@@ -0,0 +1,41 @@
+import { html } from "lit";
+import type { Meta, StoryObj } from "@storybook/web-components";
+
+const meta: Meta = {
+  title: "Components/Menu Toggle",
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+export const Open: Story = {
+  render: () => html`
+    <button class="iati-menu-toggle js-iati-menu-togle-open">
+      <svg focusable="false" viewBox="0 0 38 17" fill="currentColor">
```

Comment:
@tillywoodfield do you mean adding them as files and then interpolating  story? (as opposed to using a `<img>`).

Reply:
I was thinking similar to the ones in the `Icon` component or the country switcher, so that the user just adds a class and then it's easier for us to change upstream in the future if needed

Reply:
@tillywoodfield the advantage of embedding the SVG is that we can style the path but I'm guessing this isn't so important here. I will replace with background images but we'll lose the hover styles.

Reply:
Done @tillywoodfield

Reply:
Thank you!

It looks like the Open variant is missing a height:

![Screenshot 2024-10-01 at 10 24 29](https://github.com/user-attachments/assets/bef3c4b0-6f3d-4255-9226-395dc37e945d)

Reply:
Hi @tillywoodfield this is because when it is used in the toolbar it will stretch to fit the container height thanks to the `align-self` but obviously this doesn't work well when viewed in isolation. It may be that these buttons don't really make sense as independent components so an alternative would be to extend the header\mobile nav styles instead - any feelings on this?